demo/src/lists/lists-page.js

```diff
@@ -133,35 +132,34 @@ export class ListsPage extends LitElement {
     return this.#abortController.signal;
   }
 
-  #openPlayer(event) {
-    const button = event.target.closest('button');
-
-    if (!('urn' in button.dataset)) return;
-
-    openPlayerModal({ src: button.dataset.urn, type: 'srgssr/urn' });
+  #toMediaButtonParams(node) {
+    return new URLSearchParams({ ...router.queryParams, src: node.urn, type: 'srgssr/urn' }).toString();
   }
 
-  #renderMediaButton(node, idx) {
+  #renderMediaButton(node) {
     const date = new Intl.DateTimeFormat('fr-CH').format(new Date(node.date));
     const duration = Pillarbox.formatTime(node.duration / 1000);
 
     return html`
-        <button class="content-btn" data-node-idx="${idx}"
-                title="${node.title}">
-            <span class="content-btn-title">${node.title}</span>
-            <div class="content-btn-metadata-container">
-                <i class="material-icons-outlined">${node.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
-                <span class="content-btn-info">&nbsp;| ${date} | ${duration}</span>
-            </div>
-        </button>
+      <content-link title="${node.title}" href="lists?${this.#toMediaButtonParams(node)}">
+        <i class="material-icons-outlined">${node.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
+        <span>&nbsp;| ${date} | ${duration}</span>
```

Comment:
The icon and date/duration isn't rendered without something like
```suggestion
 <div slot="description">
        <i class="material-icons-outlined">${node.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
        <span>&nbsp;| ${date} | ${duration}</span>
</div>
```

Reply:
then maybe adding some CSS to align the icon